src/frontend/apps/desk/src/app/globals.css

```diff
@@ -4,3 +4,22 @@ body {
   margin: 0;
   padding: 0;
 }
+
+::-webkit-scrollbar {
+  width: 20px;
+}
+
+::-webkit-scrollbar-track {
+  background-color: transparent;
+}
+
+::-webkit-scrollbar-thumb {
+  background-color: #d6dee1;
+  border-radius: 20px;
+  border: 6px solid transparent;
+  background-clip: content-box;
+}
+
+::-webkit-scrollbar-thumb:hover {
+  background-color: #a8bbbf;
+}
```

Comment:
But are the width and border-radius actually intended to be the same at all times?

Reply:
It's a guess, but I am pretty these two quantities match to have a nice rounded thumb in the scroll bar. Actually, I would suggest replacing the `border-radius: 20px;`  by a large pixel size `border-radius: 9999px;` (inspiration from tailwind rounded-full class).

Reply:
I am just saying, for a developer reading the code, both quantities are linked, because the actual concept is that you want the thumb to be fully rounded, based on scrollbar width. It's not explicit reading the code. And I agree it's totally okay to not couple these values that may evolve over time. Ignore my comment